Automattic/wp-calypso
# My Home: Persist "Quick links" toggle status

#### Changes proposed in this Pull Request

Suggested by @gwwar in https://github.com/Automattic/wp-calypso/pull/41229, this PR persists the toggle status of the new "Quick links" card added to the experimental Home layout, so it remains expanded or collapsed when visiting again My Home.

#### Testing instructions

* Enable the experimental Home layout: `/home/:site?flags=home/experimental-layout`.
* Quick links should be collapsed by default.
* Expand them and reload the page.
* Make sure they remain expanded.
* Switch to a WP for Teams site.
* Make sure they remain expanded.
* Collapse them and reload the page.
* Make sure they remain collapsed.
* Switch back to the previous site.
* Make sure they remain collapsed.
